src/wsi/sdl3/wsi_platform_sdl3.cpp

```diff
@@ -0,0 +1,78 @@
+#if defined(DXVK_WSI_SDL3)
+
+#include "wsi_platform_sdl3.h"
+#include "../../util/util_error.h"
+#include "../../util/util_string.h"
+#include "../../util/util_win32_compat.h"
+
+#include <SDL3/SDL_vulkan.h>
+
+namespace dxvk::wsi {
+
+  Sdl3WsiDriver::Sdl3WsiDriver() {
+    libsdl = LoadLibraryA( // FIXME: Get soname as string from meson
+#if defined(_WIN32)
+        "SDL3.dll"
+#elif defined(__APPLE__)
+        "libSDL3.dylib"
```

Comment:
```suggestion
        "libSDL3.0.dylib"
```

Reply:
Thanks, wasn't sure what the Apple library was called.